R/data-set.R

```diff
@@ -125,33 +146,57 @@ DataSet <- R6::R6Class(
       }
       private$throwPropertyIsReadonly("yErrorValues")
     },
+
     #' @field molWeight Molecular weight of the yValues in g/mol
+
     molWeight = function(value) {
       if (missing(value)) {
         molWeight <- private$.yColumn$molWeight
         if (is.null(molWeight)) {
           return(NULL)
         }
-        return(toUnit(quantityOrDimension = ospDimensions$`Molecular weight`, values = molWeight, targetUnit = ospUnits$`Molecular weight`$`g/mol`))
+        return(toUnit(
+          quantityOrDimension = ospDimensions$`Molecular weight`,
+          values = molWeight,
+          targetUnit = ospUnits$`Molecular weight`$`g/mol`
+        ))
       }
 
-      private$.yColumn$molWeight <- toBaseUnit(quantityOrDimension = ospDimensions$`Molecular weight`, values = value, unit = ospUnits$`Molecular weight`$`g/mol`)
+      private$.yColumn$molWeight <- toBaseUnit(
+        quantityOrDimension = ospDimensions$`Molecular weight`,
+        values = value,
+        unit = ospUnits$`Molecular weight`$`g/mol`
+      )
     },
     #' @field LLOQ LLOQ Lower Limit Of Quantification.
     #' Value in yUnit associated with the yValues
+
     LLOQ = function(value) {
       if (missing(value)) {
         # Value in base unit
         lloq <- private$.yColumn$LLOQ
         if (is.null(lloq)) {
           return(NULL)
         }
-        return(toUnit(quantityOrDimension = private$.yColumn$dimension, values = lloq, targetUnit = private$.yColumn$displayUnit))
+
+        return(

```

Comment:
I think return only is weird and not consistent with the rest. See below

Reply:
I have reverted this change.